pixeltable/catalog/path.py

```diff
@@ -55,5 +55,12 @@ def is_ancestor(self, other: Path, is_parent: bool = False) -> bool:
         is_prefix = self.components == other.components[: self.len]
         return is_prefix and (self.len == (other.len - 1) or not is_parent)
 
+    def ancestors(self):
+        """
+        Return all non-root ancestors of this path.
```

Comment:
missing in comment: in what order

Also: why not include the root? Excluding it makes the semantics trickier.

Reply:
I started with including root but then was trying to think of what would be returned in ancestor() of root path.